buidler.config.js

```diff
@@ -0,0 +1,23 @@
+usePlugin('solidity-coverage');
+usePlugin('@nomiclabs/buidler-truffle5');
+
+extendEnvironment(env => {
+  const { contract } = env;
+  env.contract = function (name, body) {
+    // remove the default account from the accounts list used in tests, in order
+    // to protect tests against accidentally passing due to the contract
+    // deployer being used subsequently as function caller
+    contract(name, accounts => body(accounts.slice(1)));
```

Comment:
I love this. @alcuadrado make this the default behavior pretty please

Reply:
I'm not sure I understand this. Is it like the `contract` thing in truffle?

Reply:
This, this is a truffle plugin. I was referring to the feature of hiding the default account however.

Reply:
It's kind of opinionated, and I suppose the goal of the Truffle plugins is to have as much compatibility as possible.